testing/tests/DevExpress.ui.widgets.pivotGrid/sortable.tests.js

```diff
@@ -56,6 +56,15 @@ QUnit.testStart(function() {
 
     $('#qunit-fixture').html(markup);
 
+    $('#sortable').css({
+        height: '300px',
+        width: '300px'
+    });
+
+    $('#swatchSortable').css({
+        height: '300px',
+        width: '300px'
+    });
```

Comment:
Let's move this CSS rules to ```<style>```:

```suggestion
    #sortable {
        height: 300px;
        width: 300px;
    }

    #swatchSortable {
        height: 300px;
        width: 300px;
    }
```